data/data/agent/files/etc/motd

```diff
@@ -7,4 +7,7 @@ sudo journalctl -u agent.service
 
 To view the agent log, run:
 sudo journalctl TAG=agent
-**  **  **  **  **  **  **  **  **  **  **  **  **  **  **  **  **  ** **  **  **  **  **  **  **
\ No newline at end of file
+
+To gather essential troubleshooting data for bug reports and support, you can run:
+sudo /usr/local/bin/agent-gather
```

Comment:
Why not use sudo where required in the script? the `core` user has passwordless sudo.